pkg/portal/cluster/prometheusalerts.go

```diff
@@ -0,0 +1,46 @@
+package cluster
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+import (
+	"context"
+	"fmt"
+
+	"github.com/Azure/ARO-RP/pkg/util/namespace"
+)
+
+type FiringAlert struct {
```

Comment:
This probably could just be called alert, since we have the status inside of it

Reply:
done